pkg/storaged/stratis-details.jsx

```diff
@@ -520,7 +520,10 @@ export const StratisPoolDetails = ({ client, pool }) => {
                               }),
                     TextInput("mount_point", _("Mount point"),
                               {
-                                  validate: val => is_valid_mount_point(client, null, val)
+                                  validate: (val, values, variant) => {
```

Comment:
This only allows the empty string as the mountpoint, but the fstab entry is still written later in `set_mount_options`. That function turns a empty string into "/", actually. :)
So we also need to change `set_mount_options` to not do anything when the mount point is left empty.

Reply:
How about not calling `set_mount_options` from the start? I think putting the rest of the function (starting from line 309) into a big `if` would mess up the return values.

Reply:
I edited `set_mount_options` as it would be a more general solution